src/modules/workspaces/workspaces.usecase.spec.ts

```diff
@@ -949,4 +951,214 @@ describe('WorkspacesUsecases', () => {
       expect(assignableSpace).toBe(BigInt(300000));
     });
   });
+
+  describe('getWorkspaceMembers', () => {
+    const owner = newUser();
+    const workspace = newWorkspace({ owner });
+
+    it('When workspaceId does not exist then it should throw an error', async () => {
+      const workspaceId = 'not-exist-uuid';
+      const findWorkspace = jest.spyOn(workspaceRepository, 'findOne');
+      findWorkspace.mockReturnValueOnce(null);
+
+      expect(async () => {
+        await service.getWorkspaceMembers(workspaceId, owner);
+      }).rejects.toThrow();
+
+      expect(findWorkspace).toBeCalledWith({
+        id: workspaceId,
+        ownerId: owner.uuid,
+      });
+    });
+
+    it('When there are no members then return an empty array', async () => {
+      const workspaceUsers = jest.spyOn(
+        workspaceRepository,
+        'findWorkspaceUsers',
+      );
+      workspaceUsers.mockResolvedValue([]);
+
+      await expect(
+        service.getWorkspaceMembers(workspace.id, owner),
+      ).resolves.toStrictEqual([]);
+
+      expect(workspaceUsers).toHaveBeenCalled();
+    });
+
+    it('When a user is registered as member and is not the owner then the "isOwner" field will be False', async () => {
+      const user = newUser();
+      const workspaceUser = newWorkspaceUser({
+        workspaceId: workspace.id,
+        memberId: user.uuid,
+        member: user,
+      });
+
+      const getWorkspace = jest
+        .spyOn(workspaceRepository, 'findOne')
+        .mockResolvedValue(Promise.resolve(workspace));
```

Comment:
Why are you using promise.resolve? Please follow the previous tests here, you have a lot of previous examples.